### DISCUSSION TITLE
Need clarity on Allowed Tags

### DISCUSSION THREAD

**Discussion by pras5123** [mganss/HtmlSanitizer#540] [Q&A] [answered]
Below are the tags mentioned to be allowed by default.  But why is it removing html,body tags etc ? I wanted some clarity on the documentation of this. I am aware these must not be allowed from security perspective but why does it show that it is allowed ?     
 **Tags allowed by default**
a, abbr, acronym, address, area, article, aside, b, bdi, big, blockquote, body, br, button, caption, center, cite, code, col, colgroup, data, datalist, dd, del, details, dfn, dir, div, dl, dt, em, fieldset, figcaption, figure, font, footer, form, h1, h2, h3, h4, h5, h6, head, header, hr, html, i, img, input, ins, kbd, keygen, label, legend, li, main, map, mark, menu, menuitem, meter, nav, ol, optgroup, option, output, p, pre, progress, q, rp, rt, ruby, s, samp, section, select, small, span, strike, strong, sub, summary, sup, table, tbody, td, textarea, tfoot, th, thead, time, tr, tt, u, ul, var, wbr

**Comment by mganss** [comment, marked as answer]
Are you using the `Sanitize()` method? This is for sanitizing HTML fragments, not whole documents, resulting in the html and body tags getting removed. If you want to sanitize a whole HTML document, use the `SanitizeDocument()` method.

**Reply by pras5123** [reply]
Thanks , That helps
